src/generator/qbe.rs

```diff
@@ -767,7 +772,7 @@ impl QbeGenerator {
     /// Returns a QBE type for the given AST type
     fn get_type(&self, ty: Type) -> GeneratorResult<qbe::Type> {
         match ty {
-            Type::Any => Err("'any' type is not supported".into()),
+            Type::Any => Ok(qbe::Type::Long),
```

Comment:
Is this safe?

Reply:
Considering that QBE generator doesn't handle `any` properly, I'd prefer to keep this error for now

Reply:
> Is this safe?

Probably not safe, but I'm still learning QBE so.... skill issue. It comes from a place of mapping void* from c to type `l` in QBE, given that pointers are represented like this (eg. string).